<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sistema</class>
 <widget class="QMainWindow" name="Sistema">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>525</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>217</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>165</red>
        <green>100</green>
        <blue>112</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>217</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>165</red>
        <green>100</green>
        <blue>112</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>217</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>165</red>
        <green>100</green>
        <blue>112</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>165</red>
        <green>100</green>
        <blue>112</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Sistema</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bot_salir">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_elim">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminasion Gaussiana</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_Gaussei">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauss-Seidel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_pfij">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Iteracion de Punto Fijo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_raph">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Newton-Raphson</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_regr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Regresion Lineal Multiple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_euler">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Metodo de Euler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_inte">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Integracion Numerica</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Metodos Numericos </string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_grange">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpolacion de La Grange</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_derivacion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Derivacion númerica</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_reglin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Regresión Lineal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_runge">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Runge-Kutta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_calculadora">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculadora</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>222</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
